README.md

```diff
@@ -42,7 +42,7 @@ nats.request('help', null, {'max':1}, function(response) {
 
 
 // Request for single response with timeout.
-nats.requestOne('help', null, 1000, function(response) {
+nats.requestOne('help', null, {'max':1}, 1000, function(response) {
```

Comment:
Understand 👍 . In any case the argument is missing (null or {}). I will do a quick modification.